Quotient/e2ee/cryptoutils.h

```diff
@@ -12,11 +12,23 @@
 #include <QtCore/QString>
 
 namespace Quotient {
-struct QUOTIENT_API HkdfKeys {
-    //! @brief Key to be used for AES encryption / decryption
-    QByteArray aes;
-    //! @brief Key to be used for MAC creation / verification
-    QByteArray mac;
+
+// Common remark: OpenSSL is a private dependency of libQuotient, meaning that
+// headers of libQuotient can't include OpenSSL headers. Instead, alias
+// the value or type along with a comment (see SslErrorCode e.g.) and add
+// static_assert in the .cpp file to check it against the OpenSSL definition.
+
+constexpr auto DefaultPbkdf2KeyLength = 32u;
+constexpr auto AesKeySize = 32u;
```

Comment:
Since the key size is different for AES-128 and AES-256, we could call this `Aes256KeySize`

Reply:
If that's the only issue, I'm beyond happy ;)